src/themes/default/components/core/blocks/Auth/Login.vue

```diff
@@ -165,17 +166,10 @@ export default {
   .btn-full {
     box-sizing: border-box;
     width: 100%;
-    justify-content: center;
-    text-align: center;
-  }
-
-  .pass-container {
-    position: relative;
   }
 
   .icon {
     cursor: pointer;
```

Comment:
we should add atomic class `pointer` - it's used in many places

Reply:
Done